src/main.ts

```diff
@@ -1,5 +1,9 @@
+// Import Baklava
 import { BaklavaVuePlugin } from '@baklavajs/plugin-renderer-vue';
 import '@baklavajs/plugin-renderer-vue/dist/styles.css';
+
+import './assets/scss/style.scss'; // Our style
```

Comment:
Maybe it'd be great to use `@/assets/scss/style.scss` here and below so we can have this convention throughout the code! (`@` refers to `src`)